dracut/30ignition/ignition-files.service

```diff
@@ -6,11 +6,12 @@ Before=initrd-parse-etc.service
 Requires=initrd-root-fs.target
 After=initrd-root-fs.target
 
-#Requires=ignition-setup.service ignition-disks.service
-#After=ignition-setup.service ignition-disks.service
+#Requires=ignition-setup.service
+#After=ignition-setup.service
```

Comment:
should we just delete the above two lines?

Reply:
I'd say not for now, there are still many cleanups that we can do in a single sweep once we have stabilized the whole initramfs sequence. But if I need to respin this PR for another round I can drop them there, yes.